build.sbt

```diff
@@ -1,11 +1,24 @@
 import org.broadinstitute.monster.sbt.model.JadeIdentifier
 
-val okhttpVersion = "4.4.0"
+val okhttpVersion = "4.4.1"
 val vaultDriverVersion = "5.1.0"
 
 lazy val `dog-aging-ingest` = project
   .in(file("."))
   .aggregate(`dog-aging-hle-extraction`, `dog-aging-hle-transformation`)
+  .settings(publish / skip := true)
+
+lazy val `dog-aging-schema` = project
+  .in(file("schema"))
+  .enablePlugins(MonsterJadeDatasetPlugin)
+  .settings(
+    jadeDatasetName := JadeIdentifier
+      .fromString("broad_dsp_dog_aging")
+      .fold(sys.error, identity),
+    jadeDatasetDescription := "Mirror of the Dog Aging Project, maintained by Broad's Data Sciences Platform",
+    jadeTablePackage := "org.broadinstitute.monster.dogaging.jadeschema.table",
+    jadeStructPackage := "org.broadinstitute.monster.dogaging.jadeschema.struct"
+  )
```

Comment:
Reflecting our recent change to have the `MonsterJadeDatasetPlugin` sub-projects also publish a Docker image